py2pack/__init__.py

```diff
@@ -394,12 +394,17 @@ def generate(args):
 
 
 def fetch_data(args):
-    localfile = args.localfile
-    local = args.local
-
+    try:
```

Comment:
I don’t like using `try … except` instead of `if`. In this case, what’s wrong with `getattr(args, 'localfile', "")`?